apps/preview/nuxt/pages/showcases/Card/CardDefault.vue

```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable vuejs-accessibility/anchor-has-content -->
```

Comment:
line 9, do we need this anchor? if so, why ? @FRSgit

Reply:
Card can redirect to some page, like on https://www.walmart.com/

Info about that should be added to the documentation

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2915/commits/3f04e29c6dbe760abf8ecf5340d95d374de8c44e